packages/clay-toolbar/src/index.tsx

```diff
@@ -43,28 +40,24 @@ const ClayToolbar: React.FunctionComponent<IProps> & {
 } = ({
 	children,
 	className,
-	component,
 	inlineBreakpoint,
 	subnav,
 	...otherProps
 }: IProps) => {
+	subnav = subnav === true ? {} : subnav;
```

Comment:
By setting this, there is no need to check for boolean types or `subnav && ...` multiple times below